examples/InputWithDecorations.main.tsx

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { InputWithDecorations, Icon } from '@itwin/itwinui-react';
+import { SvgStar, SvgInfo } from '@itwin/itwinui-icons-react';
+
+export default () => {
+  return (
+    <InputWithDecorations>
+      <Icon padded>
+        <SvgInfo />
+      </Icon>
+      <InputWithDecorations.Input placeholder='Input with decorations' />
```

Comment:
this one is missing label. could you make sure all examples pass a11y?

Reply:
I think all InputWithDeco examples are passing a11y. (InputGroup is not part of this PR)
<img width="724" alt="image" src="https://github.com/iTwin/iTwinUI/assets/83585998/dc2be8da-b755-4797-8f72-35f0ee06e964">

Reply:
ah it's because "placeholder" can technically suffice as a label (but is not recommended).